test/test_spectral_ops.py

```diff
@@ -798,14 +797,13 @@ def test_stft_roundtrip_complex_window(self, device, dtype):
             # input
             (torch.randn(600, device=device, dtype=dtype),
              torch.randn(807, device=device, dtype=dtype),
-             torch.randn(12, 14, device=device, dtype=dtype),
-             torch.randn(9, 6, device=device, dtype=dtype)),
+             torch.randn(12, 60, device=device, dtype=dtype)),
             # n_fft
             (50, 27),
             # hop_length
             (None, 10),
             # pad_mode
-            ("constant",),
+            ("constant", "reflect", "circular"),
```

Comment:
What about replicate? It's OK if we don't support replicate in this PR, but then let's update the documentation to clarify which modes stft supports when given complex inputs and file a follow-up task for replicate padding.